### PR TITLE
Update NetBeans.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by ivysrono** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/585903056#L349

**Comment by J0WI** [comment]
Done

**Comment by ivysrono** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/586644503#L353

File: src/chrome/content/rules/Oracle-mismatches.xml [hunk 1]
```diff
@@ -4,12 +4,10 @@
 -->
 <ruleset name="Oracle (mismatches)" default_off="mismatched">
 
-	<!--	Akamai	-->
-	<!--	cert: forums.netbeans.org	-->
 	<target host="planetnetbeans.org" />
```
Comment (ivysrono) [inline src/chrome/content/rules/Oracle-mismatches.xml]: They may not exist now.

File: src/chrome/content/rules/NetBeans.xml [hunk 2]
```diff
@@ -1,69 +1,50 @@
...
 	<target host="edu.netbeans.org" />
-	<target host="forums.netbeans.org" />
+	<target host="fr.netbeans.org" />
+	<target host="ja.netbeans.org" />
+	<target host="nblocalization.netbeans.org" />
+	<target host="performance.netbeans.org" />
+	<target host="php.netbeans.org" />
+	<target host="platform.netbeans.org" />
+	<target host="profiler.netbeans.org" />
 	<target host="services.netbeans.org" />
```
Comment (ivysrono) [inline src/chrome/content/rules/NetBeans.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/591813141#L347

**Comment by ivysrono** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/591934294#L321
https://travis-ci.org/EFForg/https-everywhere/jobs/591934295#L485

**Comment by ivysrono** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/593025006#L513

**Pull request merged**